hadoop-ozone/dist/src/main/smoketest/recon/recon-legacy-nssummary.robot

```diff
@@ -0,0 +1,146 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This test is almost identical to the fso one.  See if you can parameterize them and invoke them twice with different parameters.  See some examples in hadoop-ozone/dist/src/main/compose/ozone/test.sh